README.md

```diff
@@ -87,6 +87,12 @@ On Linux and macOS, you can use this install script, which will automatically in
 curl -sSfL https://www.shuttle.rs/install | bash
 ```
 
+On Windows, you can use this install script to do the same:
+
+```powershell
+Set-ExecutionPolicy Unrestricted -Scope Process; iex (iwr "https://www.shuttle.rs/install-win").Content
```

Comment:
Which step is the policy change needed for? Would be nice if this line is as short as possible. Is it possible to move the policy thing into the script?

Also, is piping not a thing in PS?

Reply:
The execution policy is set for the current terminal session, which I believe was needed for Invoke-Expression (iex) to work. I don't think setting it inside the expression would work.

I can fix the pipe, oops.